src/assets/wise5/components/table/tableController.ts

```diff
@@ -276,12 +297,68 @@ class TableController extends ComponentController {
       this.dataExplorerGraphType = this.dataExplorerGraphTypes[0].value;
     }
     this.isDataExplorerScatterPlotRegressionLineEnabled = this.componentContent.isDataExplorerScatterPlotRegressionLineEnabled;
-    if (this.componentContent.numDataExplorerYAxis > 1) {
-      this.dataExplorerYAxisLabels = Array(this.componentContent.numDataExplorerYAxis).fill('');
-    }
+    this.dataExplorerYAxisLabels = Array(this.componentContent.numDataExplorerYAxis).fill('');
     this.dataExplorerSeriesParams = this.componentContent.dataExplorerSeriesParams;
   }
 
+  setDataExplorerFixedAxisToColumn(): void {
+    for (
+      let dataExplorerSeriesIndex = 0;
+      dataExplorerSeriesIndex < this.dataExplorerSeries.length;
+      dataExplorerSeriesIndex++
+    ) {
+      const xColumn = this.getDataExplorerXAxisToColumn();
+      if (xColumn != null) {
+        this.dataExplorerSeries[dataExplorerSeriesIndex].xColumn = xColumn;
+        this.dataExplorerXColumn = xColumn;
+        this.setDataExplorerXColumnIsDisabled();
+      }
+      const yColumn = this.getDataExplorerYAxisToColumn(dataExplorerSeriesIndex + 1);
+      if (yColumn != null) {
+        this.dataExplorerSeries[dataExplorerSeriesIndex].yColumn = yColumn;
+        this.dataExplorerSeries[dataExplorerSeriesIndex].name = this.columnNames[yColumn];
+        this.setDataExplorerYColumnIsDisabled(dataExplorerSeriesIndex + 1);
+      }
+    }
+  }
+
+  isAllDataExplorerSeriesSpecified(): boolean {
+    for (const singleDataExplorerSeries of this.dataExplorerSeries) {
```

Comment:
Nit: I think we should just use series here instead of singleDataExplorerSeries. It makes the code shorter and easier to read. What do you think?